### PR TITLE
version bump script

### PR DESCRIPTION
**Pull request by kmathew-pdftron**

### CODE REVIEW

File: .github/workflows/versionBump.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        run: | 
+          git fetch 
+          git checkout -m origin/master pubspec.yaml
+          git add pubspec.yaml
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git commit -m "Updating pubspec.yaml" -a || echo "No commit needed. pubspec.yaml is already up-to-date"
+               
+      - name: Install Snapcraft
+        uses: samuelmeuli/action-snapcraft@v1.2.0
```
Comment (sgong-pdftron) [inline .github/workflows/versionBump.yml:28]: who's this?

Comment (kmathew-pdftron) [reply .github/workflows/versionBump.yml:28]: it's like npm basically managing packages or snaps of software. I'm just using a GitHub action from the marketplace that easily installs snap craft into the ubuntu machine

Comment (sgong-pdftron) [reply .github/workflows/versionBump.yml:28]: @kmathew-pdftron did you have chance to document this plugin yet?

Comment (kmathew-pdftron) [reply .github/workflows/versionBump.yml]: using command line commands instead of action

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**